src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -19,10 +19,25 @@
 import subprocess
 from subprocess import Popen, PIPE, run, STDOUT, call, DEVNULL
 from azure.cli.testsdk import (LiveScenarioTest, ResourceGroupPreparer, live_only)  # pylint: disable=import-error
+from azure.cli.core.azclierror import RequiredArgumentMissingError
 
 TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
 logger = get_logger(__name__)
 
+# Set up configuration file. If configuration file is not found, then auto-populate with fake values where allowed.
+CONFIG = {} # dictionary of configurations
+config_path = os.path.join(os.path.dirname(__file__), "config.json")
+if not os.path.isfile(config_path):
+    CONFIG["customLocationsOid"] = ""
+    CONFIG["rbacAppId"] = "fakeRbacAppId"
+    CONFIG["rbacAppSecret"] = "fakeRbacAppSecret"
+    CONFIG["location"] = "eastus2euap"
+else:
+    with open(config_path, 'r') as f:
+        CONFIG = json.load(f)
```

Comment:
>   File "/mnt/vss/_work/1/s/src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py", line 29, in <module>
    with open(configPath, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/vss/_work/1/s/src/connectedk8s/azext_connectedk8s/tests/latest/config.json'

`connectedk8s/azext_connectedk8s/tests/latest/config.json` file was used in the testing of CI pipeline, why do you need to remove this file? Could you keep a config file which does not contain sensitive information to make the CI tests pass?

Reply:
No this is not possible because this is a live test. You need passwords. Luckily, the code doesn't verify that the password actually works, so I can use a fake password for now just for testing.

Reply:
> Luckily, the code doesn't verify that the password actually works, so I can use a fake password for now just for testing. 

OK, this is exactly the way I want to recommend

Reply:
> Ideally, Azure CLI CI pipeline should have a way to configure secrets for live tests, but I don't know if Azure CLI pipeline actually runs these tests, or ignore missing configurations files related to manual user configurations.

The CI pipeline will only run tests in playback mode instead of live mode. Regarding the issue of whether the CI pipeline can support configuring secrets files, add our CI expert @wangzelin007  to help take a look

Reply:
We cannot store secrets for security reasons. 
If you want run live test successfully, you should create secrets in real time, then complete your test and delete all resources immediately after the test is completed.
This way secrets only exist in memory and will no longer exist after the test is completed, which is the safest.

Reply:
@wangzelin007 This is caused by manual config file added to the live test to begin with. Azure CLI runs and doesn't see config and throws an error because a user needs to create the file. Ideally either there should be an option to ignore these config files or for Azure CLI to support secret configurations. Configurations and code should always be separate.

The work-around for now is to put in fake values because the code doesn't validate the password, but that is not a long-term solution if you have to write live tests where credentials are validated.

Reply:
Thanks, since we don't need real password for the tests currently, temporary real password feature can be implemented for another PR when the need arises. 

@zhoxing-ms Are there any other concerns?